app/controllers/api/v0/classified_listings_controller.rb

```diff
@@ -0,0 +1,79 @@
+module Api
+  module V0
+    class ClassifiedListingsController < ApiController
+      include ClassifiedListingsToolkit
+      respond_to :json
+
+      before_action :set_classified_listing, only: %i[show update]
+      before_action :authenticate!, only: %i[create update]
+
+      # skip CSRF checks for create and update
+      skip_before_action :verify_authenticity_token, only: %i[create update]
+
+      def index
+        @classified_listings = ClassifiedListing.published.
+          order("bumped_at DESC").
+          includes(:user, :organization, :taggings)
+
+        @classified_listings = if params[:category].present?
```

Comment:
Commenting here for the entire block.

I think this should have pagination, you can copy the `ArticlesController#me` https://github.com/thepracticaldev/dev.to/blob/master/app/controllers/api/v0/articles_controller.rb#L47 (maybe set a more reasonable max since it's a public route, like 100 max per page).

The reason why is not present in the frontend website is because the website is going to load listings via JS through Algolia.

The API though needs to be able to deliver all listings, in pages.